### PR TITLE
{bio}[intel/2015b] Salmon 0.5.1 w/ Python 2.7.10 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
requires ~~#2189~~ for Jellyfish easyconfig

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2df0f8bf44acc872ed69 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/3096c396c389941bffe8 for a full test report.

File: easybuild/easyconfigs/s/Salmon/Salmon-0.5.1_skip-included-Jellyfish.patch [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+* don't hardcode $CMAKE_CXX_FLAGS
+author: Kenneth Hoste (HPC-UGent)
+--- CMakeLists.txt.orig	2015-11-14 19:24:15.000000000 +0100
++++ CMakeLists.txt	2015-12-07 14:26:45.208199682 +0100
+@@ -26,7 +26,7 @@
+ 
+ ## Set the standard required compile flags
+ # Nov 18th --- removed -DHAVE_CONFIG_H
+-set (CMAKE_CXX_FLAGS "-pthread -funroll-loops -fPIC -fomit-frame-pointer -Ofast -DRAPMAP_SALMON_SUPPORT -DHAVE_ANSI_TERM -DHAVE_SSTREAM -Wall -Wno-reorder -Wno-unused-variable -std=c++11 -Wreturn-type -Werror=return-type")
++set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -L$ENV{EBROOTJELLYFISH}/lib -ljellyfish-2.0 -pthread -DRAPMAP_SALMON_SUPPORT -DHAVE_ANSI_TERM -DHAVE_SSTREAM -Wall -Wno-reorder -Wno-unused-variable -std=c++11 -Wreturn-type -Werror=return-type")
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/Salmon/Salmon-0.5.1_skip-included-Jellyfish.patch]: let EB set `-std=c++11`?

Comment (boegel) [inline easybuild/easyconfigs/s/Salmon/Salmon-0.5.1_skip-included-Jellyfish.patch]: pfft, there's little to gain from that?

Comment (wpoely86) [inline easybuild/easyconfigs/s/Salmon/Salmon-0.5.1_skip-included-Jellyfish.patch]: I will let it slide for once :smiling_imp:

**Comment by wpoely86** [comment]
lgtm

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**